class/systems.class.php

```diff
@@ -190,10 +190,37 @@ public function getSystemActivities($system)
 		$this->db->bind(':system', $system);
 		$result = $this->db->resultset();
 		$this->db->closeQuery();
-		
+
 		return $result;
 	}
 
+		/**
+	 * Get participants of currently active cache
+	 * @param unknown $system
+	 * @return string
+	 */
+	public function getActiveCacheParticipants($system)
+	{
+			$queryString = "SELECT Distinct Pilot
+					FROM activity
+					WHERE System = :system
+					AND ActivityDate >= (SELECT ActivityDate
+							FROM activity
+							WHERE System = :systemII
+							And EntryType = 'sower'
+							ORDER By ActivityDate ASC
```

Comment:
I think the ASC in the inner query is wrong. Because you get the oldest SOW activity first. You want the last sower I think.

Reply:
This should work with a single sown cache in the system only.